src/Tribe/Timezones.php

```diff
@@ -408,7 +408,7 @@ public static function is_valid_timezone( $candidate ) {
 	 */
 	public static function timezone_from_utc_offset( $utc_offset_string ) {
 		// Test for strings looking like "UTC-2" or "UTC+5.25" etc
-		if ( ! preg_match( '/^UTC[\-\+]{1}[0-9\.]{1,4}$/', $utc_offset_string ) ) {
+		if ( ! preg_match( '/^UTC[\-+][0-9.]{1,4}$/', $utc_offset_string ) ) {
```

Comment:
```suggestion
		if ( ! preg_match( '/^UTC[+-][0-9.]{1,4}$/', $utc_offset_string ) ) {
```

Dash at end of range doesn't have to be escaped: https://stackoverflow.com/questions/30398445/matching-dot-in-a-range-preg-match